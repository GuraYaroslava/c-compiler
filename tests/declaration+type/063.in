char a1[10];

char a2[10];

a1 = a2;