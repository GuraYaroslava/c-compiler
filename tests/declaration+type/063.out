[Ln: 5, Col: 5] Error: expression must be a modifiable lvalue
